src/test/test-models/good/services/normal_glm.stan

```diff
@@ -0,0 +1,37 @@
+// generated with brms 2.15.9
+functions {
+}
+data {
+  int<lower=1> N;  // total number of observations
+  vector[N] Y;  // response variable
+  int<lower=1> K;  // number of population-level effects
+  matrix[N, K] X;  // population-level design matrix
+  int prior_only;  // should the likelihood be ignored?
+}
+transformed data {
+  int Kc = K;
+  matrix[N, Kc] Xc;  // centered version of X without an intercept
+  vector[Kc] means_X;  // column means of X before centering
+  for (i in 1:K) {
+    means_X[i] = mean(X[, i]);
+    Xc[, i] = X[, i] - means_X[i];
+  }
+}
+parameters {
+  vector[Kc] b;  // population-level effects
+  real Intercept;  // temporary intercept for centered predictors
+  real<lower=0> sigma;  // dispersion parameter
+}
+model {
+  // likelihood including constants
+  target += normal_id_glm_lupdf(Y | X, Intercept, b, sigma);
+  // priors including constants
+  target += normal_lupdf(Intercept | 0, 3);
+  target += normal_lupdf(b | 0, 3);
+  target += normal_lupdf(sigma | 1, 2);
+}
+
+generated quantities {
+  // actual population-level intercept
+  real b_Intercept = Intercept - dot_product(means_X, b);
+}
```

Comment:
[question] should this have a newline at end of 37?  Stan's parser doesn't care, it's just annoying to look at the - in GitHub!